index.html

```diff
@@ -126,187 +846,604 @@ <h3>
-            readonly attribute WritableStream writable;
-          };
+
+          The {{WritableStreamDefaultWriter/write()}} method returns a {{Promise}} which
+          resolves when data has been written. While having some data available in
+          the transmit buffer is important to maintain good throughput awaiting this
+          {{Promise}} before generating too many chunks of data is a good practice
+          to avoid excessive buffering.
+        </aside>
+
+        The steps to <dfn data-lt="initialize udp writable stream">initialize the UDP writable stream</dfn> are:
+
+        <ol>
+          <li>
+            Let |stream:WritableStream| be a [=new=] {{WritableStream}}.
+          <li>
+            Let |signal:AbortSignal| be |stream|'s [=WritableStream/signal=].
+          <li>
+            Let |writeAlgorithm| be the following steps, given |chunk|:
+            <ol>
+              <li>
+                Let |promise:Promise| be [=a new promise=].
+              <li>
+                Assert: |signal| is not [=AbortSignal/aborted=].
+              <li>
+                Let |message:UDPMessage| be a new {{UDPMessage}}.
+              <li>
+                If |chunk| cannot be [=converted to an IDL value=] of type
+                {{UDPMessage}}, reject |promise| with a {{TypeError}} and return
+                |promise|. Otherwise, save the result of the conversion in
+                |message|.
+              <li>
+                [=In parallel=], run the following steps:
+                <ol>
+                  <li>
+                    Invoke the operating system to send |message|["{{UDPMessage/data}}"] to the socket.
+
+                    <div class="note">
+                      The |message|["{{UDPMessage/remoteAddress}}"] and |message|["{{UDPMessage/remotePort}}"]
+                      currently have no effect on sending packets since the UDP socket is only allowed to operate in the connected mode.
+                    </div>
+
+                    <div class="note">
+                      The operating system may return from this operation once
+                      |bytes| has been queued for transmission rather than after it
+                      has been transmitted.
+                    </div>
+                  <li>
+                    [=Queue a global task=] on the [=relevant global object=] of
+                    [=this=] using the [=UDP socket task source=] to run the
+                    following steps:
+                    <ol>
+                      <li>
+                        If the data was successfully written, [=resolve=] |promise| with `undefined`.
+
+                        <div class="note">
+                          [[STREAMS]] specifies that |writeAlgorithm| will only be
+                          invoked after the {{Promise}} returned by a previous
+                          invocation of this algorithm has resolved.
+                        </div>
```

Comment:
This comment isn't really relevant for UDP since we aren't combining multiple packets into a single write.

Reply:
Fixed!